pkg/build/nodeimage/const_cni.go

```diff
@@ -46,6 +46,12 @@ rules:
     verbs:
       - list
       - watch
+  - apiGroups:
```

Comment:
we don't need to be able to get _all_ configmaps..?

Reply:
done